test-runner/src/cli.ts

```diff
@@ -17,7 +17,7 @@
 import program from 'commander';
 import * as fs from 'fs';
 import * as path from 'path';
-import { collectTests, runTests, RunnerConfig } from '.';
+import { run, RunnerConfig } from '.';
```

Comment:
Because requiring index file registers fixtures and has side effects, I would expect no internal code to require it.

Reply:
Cli is not exactly internal - it is one of the runner front-ends. We could have two entry points though.